src/index.tsx

```diff
@@ -1,17 +1,19 @@
-import 'global-shim';
-import 'init-sentry';
-import React from 'react';
+import React, { Suspense } from 'react';
 import ReactDOM from 'react-dom/client';
-import 'utils/buffer';
-import 'fonts.css';
-import 'index.css';
-import { App } from 'App';
 import reportWebVitals from 'reportWebVitals';
+import { App } from 'App';
+import { StoreProvider } from 'store';
 import { Web3ReactWrapper } from 'libs/web3';
 import { Router } from 'libs/routing';
 import { LazyMotion } from 'libs/motion';
 import { QueryProvider } from 'libs/queries';
-import { StoreProvider } from 'store';
+import { CarbonLogoLoading } from 'components/common/CarbonLogoLoading';
+import './languages/i18n';
+import 'global-shim';
```

Comment:
why are you reordering these imports? I am pretty sure there was a reason that global shim and sentry is at the top.